packages/store/src/store.ts

```diff
@@ -21,7 +21,7 @@ export class Store {
   /**
    * Selects a slice of data from the store.
    */
-  select<T>(selector: (state: any) => T): Observable<T>;
+  select<T>(selector: (state: any, states?: any[]) => T): Observable<T>;
```

Comment:
This does not seem to match the signature  executed by the selector above.
Shouldn't it rather be: `selector: (state: any, ...states?: any[]) => T)`

Reply:
#facepalm